src/traces/scattermapbox/attributes.js

```diff
@@ -26,6 +26,58 @@ module.exports = overrideAll({
     lon: scatterGeoAttrs.lon,
     lat: scatterGeoAttrs.lat,
 
+    cluster: {
+        enabled: {
+            valType: 'boolean',
+            role: 'info',
+            dflt: false,
+            editType: 'plot',
```

Comment:
Please remove this line:
```js
editType: 'plot',
```
For all these attributes, the `editType` is going to be set to `calc` down below.